<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanDetail</class>
 <widget class="QDialog" name="PlanDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>计划内容：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="content">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>开始时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="BTdateTimeEdit">
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>提醒时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="RTdateTimeEdit">
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>结束时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="ETdateTimeEdit">
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>重复：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="onceRBtn">
         <property name="text">
          <string>提醒一次</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="weekRBtn">
         <property name="text">
          <string>每周</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="monthRBtn">
         <property name="text">
          <string>每月</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>生成规则：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rule">
         <property name="text">
          <string>once</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="accept">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reject">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QCheckBox" name="c1">
       <property name="text">
        <string>周一</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c2">
       <property name="text">
        <string>周二</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c3">
       <property name="text">
        <string>周三</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c4">
       <property name="text">
        <string>周四</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c5">
       <property name="text">
        <string>周五</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c6">
       <property name="text">
        <string>周六</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c7">
       <property name="text">
        <string>周日</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="monthLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
